packages/web/babel.ts

```diff
@@ -48,6 +48,7 @@ module.exports = function (api: any, options: NxReactBabelPresetOptions = {}) {
         require.resolve('@babel/plugin-proposal-class-properties'),
         options.classProperties ?? { loose: true },
       ],
+      require.resolve('babel-plugin-transform-typescript-metadata'),
```

Comment:
This needs to be before `@babel/plugin-proposal-decorators` according to the docs.